sql-statements/sql-statement-show-stats-histograms.md

```diff
@@ -6,28 +6,23 @@ aliases: ['/docs/dev/sql-statements/sql-statement-show-histograms/']
 
```

Comment:
As this file is renamed, please add the following alias to L4 '/tidb/dev/sql-statement-show-histograms'